test/pack_test.cpp

```diff
@@ -559,97 +562,102 @@ const ItemPack PackedHellfireItems[] = {
 	// clang-format on
 };
 
+#define GnatStingSpecialEffect (ItemSpecialEffect::MultipleArrows | ItemSpecialEffect::QuickAttack)
+#define ThunderclapSpecialEffect (ItemSpecialEffect::FireDamage | ItemSpecialEffect::LightningDamage)
+#define FlambeauSpecialEffect (ItemSpecialEffect::FireArrows | ItemSpecialEffect::LightningArrows)
+#define BlitzenSpecialEffect (ItemSpecialEffect::FireArrows | ItemSpecialEffect::LightningArrows)

```

Comment:
```suggestion
constexpr ItemSpecialEffect GnatStingSpecialEffect = ItemSpecialEffect::MultipleArrows | ItemSpecialEffect::QuickAttack;
constexpr ItemSpecialEffect ThunderclapSpecialEffect = ItemSpecialEffect::FireDamage | ItemSpecialEffect::LightningDamage;
constexpr ItemSpecialEffect ExplosiveArrows = ItemSpecialEffect::FireArrows | ItemSpecialEffect::LightningArrows;
```

Could even rename the constant for thunderclap to "ExplosiveDamage".